src/components/beersList/BeersListItem/BeersListItem.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import OpenDetailsPageButton from 'components/common/OpenDetailsPageButton/OpenDetailsPageButton';
+import FavouriteButton from 'components/common/FavouriteButton/FavouriteButton';
+import localStorageHelper from 'helpers/localStorageHelper';
+import favouriteItemHelper from 'helpers/favouriteItemHelper';
+
+import './beersListItem.scss';
+
+
+export default class BeersListItem extends React.PureComponent {
+    static propTypes = {
+        beer: PropTypes.shape({
+            id: PropTypes.number.isRequired,
+            imageUrl: PropTypes.string.isRequired,
+            tagline: PropTypes.string.isRequired,
+            name: PropTypes.string.isRequired,
+            description: PropTypes.string.isRequired
+        }).isRequired
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = { isFavourite: favouriteItemHelper.isFavourite(this.props.beer) };
+    }
+
+    toggleFavouriteState = () => {
+        this.state.isFavourite
+            ? localStorageHelper.deleteItem(this.props.beer)
+            : localStorageHelper.add(this.props.beer);
+        this.setState({ isFavourite: !this.state.isFavourite });
+    }
+
+    render() {
+        const { beer } = this.props;
+        return (
+            <div className="beers-list-item">
+                <img alt="Beer" src={beer.imageUrl} className="beers-list-item__image" />
+                <div className="beers-list-item__information">
+                    <div className="beers-list-item__title">{beer.name}</div>
+                    <div className="beers-list-item__tagline">{beer.tagline}</div>
+                    <OpenDetailsPageButton beerID={beer.id} className="open-details-page-button--on-beers-list" />
+                    <FavouriteButton beer={beer} className="favourite-button--on-beers-list" />
```

Comment:
You don't have this style in `BeersListItem` component. The same for other components that use `FavouriteButton`.
Just remove them in button, move them to the appropriate components, and give them correct names.